### PR TITLE
Add LReLU parameterization

### PR DESCRIPTION
**Pull request by Randl**
- [x] Fix deserialization bug

### CODE REVIEW

**Comment by Randl** [comment]
@edgarriba, I have a problem with deserialization, so your approval is a bit early ;)

File: tiny_dnn/util/serialization_functions.h [hunk 1]
```diff
@@ -291,8 +291,10 @@ struct LoadAndConstruct<tiny_dnn::leaky_relu_layer> {
   static void load_and_construct(
     Archive &ar, cereal::construct<tiny_dnn::leaky_relu_layer> &construct) {
     tiny_dnn::shape3d in_shape;
+    tiny_dnn::float_t epsilon;
 
-    ar(cereal::make_nvp("in_size", in_shape));
+    ar(cereal::make_nvp("in_size", in_shape),
+       cereal::make_nvp("epsilon", epsilon));
     construct(in_shape);
```
Comment (beru) [inline tiny_dnn/util/serialization_functions.h]: New argument `epsilon` should be supplied to `construct` method/function?.
```cpp
     construct(in_shape, epsilon);
```

File: test/test_serialization.h [hunk 2]
```diff
@@ -442,6 +443,7 @@ TEST(serialization, serialize_leaky_relu) {
   EXPECT_EQ(net[0]->layer_type(), "leaky-relu-activation");
   EXPECT_EQ(net[0]->in_shape()[0], shape3d(256, 256, 1));
   EXPECT_EQ(net[0]->out_shape()[0], shape3d(256, 256, 1));
+  EXPECT_FLOAT_EQ(net.at<leaky_relu_layer>(0).epsilon_value(), float_t{0.1});
```
Comment (beru) [inline test/test_serialization.h]: This should be
```cpp
   EXPECT_FLOAT_EQ(net.at<leaky_relu_layer>(0).epsilon_value(), float_t(0.1));
```

**Comment by edgarriba** [comment]
@Randl ping once you are done so that we can merge

**Comment by Randl** [comment]
@edgarriba I've fixed it

**Pull request merged**